templates/default/chef-server-image.erb

```diff
@@ -80,13 +82,16 @@ if [ -z "${email}" ];then
 fi
 
 if [ -z "${password}" ];then
-  echo "Enter secure password for user:"
-  read password
+  while read -s -p 'Enter new password: ' password && [[ -z "$password" ]] ; do
```

Comment:
Should we have a limit here? e.g. Prompt user for password 3 times and if user doesn't provide then exit.